esp/esp/users/models/__init__.py

```diff
@@ -1473,7 +1473,7 @@ class TeacherInfo(models.Model, CustomFormsLinkModel):
         ('shirt_type', 'Shirt type'),
```

Comment:
For the teacherinfo field section, I dont understand this comment:

![image](https://user-images.githubusercontent.com/25392075/118063221-a323ca80-b366-11eb-8110-35edfffd9b8b.png)

My main thing is I don't see any '*'s anywhere, so I'm just confused by it

Reply:
I see them now for contact info. However, I don't really believe this comment. Having a custom form doesnt mean that users would not have to update their profile info, so why should we be including them in the custom form?

Reply:
I think what it is trying to say is that the fields need to be required if you include them in the form, because they are required fields in the profile? But there was a bug that didn't set the required checkbox. I fixed the bug and I changed the text of the comment too.